### ISSUE TITLE
AJAX settings are not accesible and have other isssues.

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#1047]
https://github.com/CalderaWP/Caldera-Forms/blob/72327f9f111af7d70f9cb6b0ce68f6b02d27edec/includes/ajax.php#L36-L89

- [x] Missing for on labels
- [x] Checkboxes should use fieldset/legend
- [x] Extra space in this description `  If set, form can be submitted multiple times with out a new page load.`
- [x] Missing escape for some translation strings
- [x] Needs aria-describedy for descriptions

**Comment by Shelob9** [comment]
Close via 		d784a6c
